openid-4-verifiable-presentations-1_0.md

```diff
@@ -3204,6 +2944,8 @@ The technology described in this specification was made available from contribut
    * clarify that `dcql_query` and `presentation_definition` are passed as JSON objects (not strings) in request objects
    * support returning multiple presentations for a single dcql credential query when requested using `multiple`
    * Added support for multiple Client Identifiers and corresponding Request Signature to the DC API profile
+   * remove DIF Presentation Exchange as a query language option
+   * Changes in the DCQL query parameters specific to W3C VCs and AnonCreds
```

Comment:
These should go against -26 instead of -25

Reply:
Thanks, fixed after merging main.